examples/flutter/objectbox_demo_desktop/lib/main.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/foundation.dart' show debugDefaultTargetPlatformOverride;
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import "package:objectbox/objectbox.dart";
+part "main.g.dart";
+
+@Entity(id: 1, uid: 1)
+class Note {
+    @Id(id: 1, uid: 1001)
+    int id;
+
+    @Property(id: 2, uid: 1002)
+    String text;
+
+    Note();
+    Note.construct(this.text);
+    toString() => "Note{id: $id, text: $text}";
+}
+
+void main() {
+    // See https://github.com/flutter/flutter/wiki/Desktop-shells#target-platform-override
+    debugDefaultTargetPlatformOverride = TargetPlatform.fuchsia;
+
+    runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+    @override
+    Widget build(BuildContext context) {
+        return MaterialApp(
+            title: "ObjectBox Demo",
+            theme: ThemeData(primarySwatch: Colors.blue, fontFamily: "Roboto"),
+            home: MyHomePage(title: "ObjectBox Demo"),
+        );
+    }
+}
+
+class MyHomePage extends StatefulWidget {
+    MyHomePage({Key key, this.title}) : super(key: key);
+
+    final String title;
+
+    @override
+    _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+    String _status = "";
+    Store _store;
+    Box _box;
+    int _lastPutId;
+
+    void _testGet() {
+        setState(() {
+            if(_lastPutId == null) {
+                _status += "cannot get, as nothing was put in this session yet\n";
+                return;
+            }
+            _status += "fetched note: ${_box.get(_lastPutId)}\n";
+        });
+    }
+
+    void _testPut() {
+        setState(() {
+            _lastPutId = _box.put(Note.construct("Hello"));
```

Comment:
I know little about Flutter, but I was in 1 talk at droidcon about 60FPS in Flutter 😀: and the guy said inside `setState()` no computation should be done, only mutation (ok, [the docs](https://api.flutter.dev/flutter/widgets/State/setState.html) say this, too). So put the note outside of `setState()` and only assign the given ID inside?

Probably applies to the code in `_testGet()` as well.

Reply:
Yep, makes sense; fixed it.